index.js

```diff
@@ -11,3 +12,9 @@ function Store (dbName, options) {
 
   this.db = new PouchDB(dbName, options)
 }
+
+Store.prototype.add = function (objects) {
+  var api = this.db.hoodieApi()
+
+  return api.add(objects)
+}
```

Comment:
Instead of making one prototype method per api method (and calling this.db.hoodieApi() everytime), we could simply do this

```js
function Store (dbName, options) {
  if (!(this instanceof Store)) return new Store(dbName)
  if (typeof dbName !== 'string') throw new Error('Must be a valid string.')
  options = options || {}

  var db = new PouchDB(dbName, options)
  var api = db.hoodieApi()

  return api
}
```